packages/lesswrong/server/callbacks/userCallbacks.tsx

```diff
@@ -230,6 +215,16 @@ getCollectionHooks("Users").editSync.add(async function usersEditCheckEmail (mod
       modifier.$set.emails = [{address: newEmail, verified: false}];
       await sendVerificationEmailConditional(user)
     }
+
+    if (isEAForum) {
```

Comment:
This is an existing issue, but do we have some `hasAuth0` that we could use here instead in case any other forums end up using auth0. This closest thing I know is
```
  const hasAuth0 = !!(auth0ClientId && auth0OAuthSecret && auth0Domain);
```
in authentication middlewares, but it's function-local. Maybe we could hoist it to a global and use it here.